Tools/autotest/common.py

```diff
@@ -10,6 +10,28 @@
 # messages. This keeps the output to stdout flowing
 expect_list = []
 
+class AutoTestTimeoutException(Exception):
+    pass
+
+def wait_ready_to_arm(mav, timeout=None):
+    # wait for EKF checks to pass
+    return wait_ekf_happy(mav, timeout=timeout)
+
+def wait_ekf_happy(mav, timeout=30):
+    """Wait for EKF to be happy"""
+
+    tstart = get_sim_time(mav)
+    required_value = 831
+    print("Waiting for EKF value %u" % (required_value))
+    while timeout is None or get_sim_time(mav) < tstart + timeout:
+        m = mav.recv_match(type='EKF_STATUS_REPORT', blocking=True)
+        current = m.flags
+        print("Wait EKF.flags: required:%u current:%u" % (required_value, current))
```

Comment:
We should either don't print this message or at least rate limit it. Just in our short Travis autotest job it got 337 instances of this message.